src/main/java/edu/cornell/kfs/sys/batch/RunKimFeedStep.java

```diff
@@ -21,32 +22,42 @@ public class RunKimFeedStep extends AbstractStep {
     private static final String DELTA_RUN_TYPE = "delta";
 
     private Properties kimFeedBaseProperties;
+    private KimFeedService kimFeedService;
 
     @Override
     public boolean execute(String jobName, Date jobRunDate) throws InterruptedException {
         try {
-            LOG.info("execute: Starting run of KIM feed");
+            LOG.info("execute: Starting run of legacy KIM feed");
```

Comment:
I agree that a separation between the legacy feed and the new feed makes sense, but I'm wondering whether we should have two separate jobs or if we should just have two separate steps within the same job (one for the legacy feed and one for the new feed). With two jobs, it will be more straightforward to retire the old feed when we finally retire Cynergy, but then we'd be creating a new batch job (and its Tidal setup) that will only have a short lifespan. If we have one job with two steps, we don't have to create a separate job, but we'd have to revise the existing job accordingly when the Cynergy retirement comes.

Thoughts?

Reply:
As per Sandy's email response, I will proceed with the two-jobs approach, because it would be cleaner and it only requires minimal Tidal-side work.

Reply:
Yes, I agree. Thank you!